diff --git a/node_modules/async-function/index.js b/node_modules/async-function/index.js
index e784f64..e17ee50 100644
--- a/node_modules/async-function/index.js
+++ b/node_modules/async-function/index.js
@@ -1,8 +1,5 @@
 'use strict';
 
-// eslint-disable-next-line no-extra-parens, no-empty-function
-const cached = /** @type {import('.').AsyncFunctionConstructor} */ (async function () {}.constructor);
-
 /** @type {import('.')} */
-module.exports = () => cached;
+module.exports = require('./legacy');
 
