diff --git a/node_modules/generator-function/index.js b/node_modules/generator-function/index.js
index b2e152c..e17ee50 100644
--- a/node_modules/generator-function/index.js
+++ b/node_modules/generator-function/index.js
@@ -1,8 +1,5 @@
 'use strict';
 
-// eslint-disable-next-line no-extra-parens, no-empty-function
-const cached = /** @type {GeneratorFunctionConstructor} */ (function* () {}.constructor);
-
 /** @type {import('.')} */
-module.exports = () => cached;
+module.exports = require('./legacy');
 
